src/circle.rs

```diff
@@ -0,0 +1,117 @@
+use super::generators::{IndexedPolygon, SharedVertex};
+use super::Polygon::PolyTri;
+use super::{Polygon, Triangle};
+use std::f32::consts::PI;
+use Vertex;
+
+/// Represents a circle in the XY plane with radius of 1, centered at (0, 0, 0)
+#[derive(Clone, Copy)]
+pub struct CircleU {
```

Comment:
why `U`?

Reply:
Remnant of copying SphereUV. I'll remove it.